scripts/agent/confluence.sh

```
@@ -0,0 +1,1199 @@
+}
+
+# --- Command: get-comments ---------------------------------------------------
+get_comments() {
+  local page_id="$1"
+
+  local response
+  response=$(curl -sf "${CURL_AUTH_ARGS[@]}" \
+    "${CONFLUENCE_BASE_URL}/rest/api/content/${page_id}/child/comment?expand=body.storage,version,extensions.inlineProperties&limit=50")
+
+  # Extract comments with author and body text
+  echo "$response" | python3 - <<'PYEOF'
```

Shellcheck (0.11.0)

[error] This redirection overrides piped input. To use both, merge or pass filenames.

(SC2259)